libc/test/src/math/smoke/powf_test.cpp

```diff
@@ -32,7 +32,33 @@ TEST_F(LlvmLibcPowfTest, SpecialNumbers) {
     if (!__r.success)
       continue;
 
+    // pow( sNaN, exponent)
```

Comment:
Nit: formatting inconsistency.

```suggestion
    // pow( sNaN, exponent )
```